ContactManger/src/main/java/Manager.java

```diff
@@ -0,0 +1,45 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Manager {
+    private List<Contact> myFriends;
+    int friendsCount;
```

Comment:
@anamariaTapia Is really necessary to have friends count as attribute of the class, you can retrieve this value from the size of the list